### DISCUSSION TITLE
雹 Hail v1.8.1

### DISCUSSION THREAD

**Discussion by aistra0528** [aistra0528/Hail#210] [Announcements]
- 优化应用界面，感谢 @Jesse205
- 修复 `Shizuku - 暂停` 在 Android 14 2024年3月更新后无法使用
- 修复固定快捷方式标签解冻
- 修复图标包加载
- 新增白俄罗斯语，感谢 @kilimov25
- 新增世界语翻译，感谢 @adproqwq
- 新增粤语（繁体）翻译，感谢 @adproqwq
- 更新阿拉伯语翻译，感谢 @Ab-Mr
- 更新印度尼西亚语翻译，感谢 @prasetyodedy
- 更新葡萄牙语（巴西）翻译，感谢 @Suburbanno
- 更新俄语翻译，感谢 @kilimov25
- 更新西班牙语翻译，感谢 @gallegonovato
- 更新土耳其语翻译，感谢 @oersen
- 更新乌克兰语翻译，感谢 @kilimov25
---
- UI improvements, thanks to @Jesse205
- Fix `Shizuku - Suspend` not working after March update on Android 14
- Fix pin shortcuts tag unfreezing
- Fix icon pack loading
- Add Belarusian translation, thanks to @kilimov25
- Add Esperanto translation, thanks to @adproqwq
- Add Yue (Traditional) translation, thanks to @adproqwq
- Update Arabic translation, thanks to @Ab-Mr
- Update Indonesian translation, thanks to @prasetyodedy
- Update Portuguese (Brazil) translation, thanks to @Suburbanno
- Update Russian translation, thanks to @kilimov25
- Update Spanish translation, thanks to @gallegonovato
- Update Turkish translation, thanks to @oersen
- Update Ukrainian translation, thanks to @kilimov25

<hr /><em>This discussion was created from the release <a href='https://github.com/aistra0528/Hail/releases/tag/v1.8.1'>雹 Hail v1.8.1</a>.</em>

**Comment by rahaaatul** [comment]
@aistra0528 Any plans to support work apps from main profile?

**Reply by aistra0528** [reply]
I don't use work profile, but pr is welcome!

**Reply by Yoly114514** [reply]
It's really useful

**Comment by lllshuai** [comment]
大佬，可否考虑一下类似炼妖壶的那种，解冻某APP后在**状态栏可以单独冻结**的功能，我APP好多的，有些白名单的APP需要单独冻结，分类又很不方便。 可以单独冻结能解决我的大问题！就不用每次先全部冻结再打开被误伤的APP啦。
还有，绿色守护中的，**媒体播放时不冻结**，对听歌听小说的兄弟们特别的有用。
有了这两个功能，几乎可以不用手动去批量冻结APP啦。

**Reply by lllshuai** [reply]
![微信图片_20240624145424](https://github.com/aistra0528/Hail/assets/105485888/d7643b95-4616-45b5-ab2f-02d965e1697b)

或者有一个更优的办法，在主屏幕快捷方式中增加一项“冻结前台APP”！  这样我们可以用例如“One hand operation”之类的手势APP一键冻结当前显示的APP。这样的话“雹”直接就可以封神了！！！
目前主屏幕快捷方式中做了所有APP的单独解冻和批量冻结的选项，但是可用性还是很局限，我想“雹”现在最缺的是一种快速冻结某一个APP的交互。

**Comment by constant2854** [comment]
请问小米澎湃os无法冻结系统应用能解决吗？

**Comment by Yoly114514** [comment]
萌新，看不懂操作怎么办？(滑稽笑)

**Reply by drakeyoung** [reply]
去酷安看看，里面有教程

**Reply by Yoly114514** [reply]
我开玩笑的
